x-pack/plugins/enterprise_search/README.md

```diff
@@ -0,0 +1,25 @@
+# Enterprise Search
+
+## Overview
+
+This plugin's goal is to provide a Kibana user interface to the Enterprise Search solution's products (App Search and Workplace Search). In its current MVP state, the plugin provides a basic engines overview from App Search with the goal of gathering user feedback and raising product awareness.
+
+## Development
+
+1. When developing locally, Enterprise Search should be running locally alongside Kibana on `localhost:3002`.
+2. Update `config/kibana.dev.yml` with `enterpriseSearch.host: 'http://localhost:3002'`
+3. For faster QA/development, run Enterprise Search on `elasticsearch-native` auth and log in as the `elastic` superuser on Kibana.
```

Comment:
```suggestion
3. For faster QA/development, run Enterprise Search on [elasticsearch-native auth](https://www.elastic.co/guide/en/app-search/current/security-and-users.html#app-search-self-managed-security-and-user-management-elasticsearch-native-realm) and log in as the `elastic` superuser on Kibana.
```

Reply:
My only concern is that this is an App Search-specific link. Do we have a generic Enterprise Search link? This will be a shared readme as seen here: https://github.com/elastic/kibana/pull/70979/commits/d40003b74020c6803b74a4353ced5c21394a44b0

Reply:
We do not have a generic Enterprise Search link but I'm hoping to make one whenever I have downtime (so sometime in the next decade, you know).

More context on this & a discussion with Chris here: https://github.com/constancecchen/kibana/pull/18#discussion_r441198381

Reply:
Good point Scotty

Reply:
Workplace Search MVP won't be in 7.9, right?

Reply:
It's going to be tight but maybe/we're shooting for it

Reply:
FWIW this is just an internal README so I'm definitely okay adding the link for now for context and changing later (although I also assume most of the devs running this will be Enterprise Search devs who know how to switch auth modes, so maybe a wash either way)